packages/core/src/components/status-label/status-label.css

```diff
@@ -43,3 +43,7 @@
   --status-label-background: var(--color-error);
   --status-label-color: var(--color-white);
 }
+
+.hds-status-label-rounded-corners {
```

Comment:
I think for modifier we would want double dash: `.hds-status-label--rouded-corners`